drp.py

```diff
@@ -325,17 +325,17 @@ def parse_git_url(url):
         return None
 
 
-def get_git_url(window):
-    for folder in window.folders():
-        url = None
-        try:
-            url = subprocess.check_output(["git", "-C", folder, "remote", "get-url", "origin"], universal_newlines=True)
-        except:
-            url = get_git_url_from_config(folder)
+def get_git_url(entity):
+    folder = os.path.dirname(entity)
+    url = None
+    try:
+        url = subprocess.check_output(["git", "-C", folder, "remote", "get-url", "origin"], universal_newlines=True, creationflags=subprocess.CREATE_NO_WINDOW)
```

Comment:
This flag was added in Python 3.7 and the default plugin host in ST uses 3.3, so it will never work.

You'll need to either use the workaround that is also in place in `Default/exec.py` to set the options directly (for Windows only) or tell ST to use the 3.8 host by adding a `.python-version` file in the package root with the contents `3.8`, which will also break compatibility with ST <4.

Reply:
Ah whoops, I'll change it to use STARTUPINFO for Windows for the popen options.

Reply:
Yeah, that should work for all versions (even ST2 :wink:).